client/dashboard/app/router.tsx

```diff
@@ -129,6 +130,8 @@ const siteOverviewRoute = createRoute( {
 	path: '/',
 	loader: async ( { params: { siteSlug }, preload } ) => {
 		const site = await queryClient.ensureQueryData( siteBySlugQuery( siteSlug ) );
+		// Ensure storage specifically is loaded because the warning notice can cause a layout shift
+		await queryClient.ensureQueryData( siteMediaStorageQuery( site.ID ) );
```

Comment:
I'm torn and not totally sure about this... currently, sometimes clicking on a site row is already slow because the site endpoint slow sometimes. I think layout shift for some minority sites is a reasonable tradeoff so that it won't hurt majority cases 🫠 (also it will only happen when the cache is empty)

Reply:
I hate layout shift with a burning passion 😅 
If we don't feel we can afford to pre-load the remaining storage then I would rather we switch to a new design which doesn't cause the layout to shift.

Reply:
Can we put the `await` at the end so that the Promises in the `preload` block here have chance to run?

Reply:
I've moved the `await` so that preloading starts before we block for the storage request ea497ed5f5be9859d59f366c233360109b1b52c5.

I'm still not sure whether this is better, since maybe we'd want there to be less network contention while we're running the blocking request 🤔 We've probably got multiple simultaneous connections going to `public-api` though, so probably not a problem.